controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/BuildService.java

```diff
@@ -21,37 +21,47 @@
     boolean isRunning(BuildJob buildJob);
 
 
-    // TODO jvenstad: Implement with DeploymentTrigger.Job
     class BuildJob {
```

Comment:
These are created outside the DeploymentTrigger to such an extent I went with a class for now. I'll see if I can make this an interface later.